2017/neural-networks/spanish/auto_generated.srt

```diff
@@ -19,1158 +19,1154 @@ Y quiero que te tomes un momento para apreciar lo loco
 que es que los cerebros puedan hacer esto sin esfuerzo.
 
 6
-00:00:19,700 --> 00:00:23,435
-Quiero decir, esto, esto y esto también son reconocibles como 3, 
+00:00:19,700 --> 00:00:23,311
+Es decir, esto, esto y esto también son reconocibles como 3s, 
 
 7
-00:00:23,435 --> 00:00:28,320
-aunque los valores específicos de cada píxel son muy diferentes de una imagen a otra.
+00:00:23,311 --> 00:00:28,320
+aunque los valores específicos de cada píxel sean muy diferentes de una imagen a otra.
 
 8
-00:00:28,900 --> 00:00:33,085
-Las células sensibles a la luz particulares de tu ojo que se activan cuando 
+00:00:28,900 --> 00:00:33,031
+Las células sensibles a la luz concretas de tu ojo que se disparan cuando 
```

Comment:
I find both "particulares" and "concretas" weird here. Does "particular" in the original script mean something like "specific"? If so, I think it would be fine to just drop the adjective in Spanish (to make it sound natural, the sentence would have to get longer than necessary). 

While "disparan" is more literal, I feel "activan" is more accurate (though someone more familiar with eye-related terminology could correct me if I am wrong):

```suggestion
Las células sensibles a la luz de tu ojo que se activan cuando 
```